packages/metacoin/test/utils/coverage.ts

```diff
@@ -15,6 +15,7 @@ export const coverage = {
     },
     _getCoverageSubprovider(): CoverageSubprovider {
         const defaultFromAddress = devConstants.TESTRPC_FIRST_ADDRESS;
-        return new CoverageSubprovider(config.artifactsDir, config.contractsDir, defaultFromAddress);
+        const zeroExArtifactsAdapter = new ZeroExArtifactAdapter(config.artifactsDir, config.contractsDir);
```

Comment:
Do we want to call this `ZeroExArtifact`? Why not `SolCompilerArtifact`? That is what it really it. Has nothing to do with 0x... Or do we think that `SolCompiler` is too generic?